docs/guide/image_editor.md

```diff
@@ -0,0 +1,82 @@
+# Configuring the Image Editor
+
+## Introduction
+
+When a Content Item contains Fields of the [ezimage](../api/field_type_reference.md#image-field-type) or [ezimageasset](../api/field_type_reference.md#imageasset-field-type) type, users can perform basic image editing functions with the Image Editor.
+For more information, see the [user documentation](https://doc.ibexa.co/projects/userguide/en/master/editing_images/).
+
+## Configuration
+
+You can modify the default settings to change the appearance or behaviour of the Image Editor.
+You can also expand the default set of parameters to create buttons that may be required by custom features that you add by [extending the Image Editor](#extending-the-image-editor).
+To do this, modify the `config/packages/ezplatform.yaml` file, or create a separate YAML file in the `config/packages` folder, and add a settings tree similar to the following example.
+Image Editor settings are [dynamic](config_dynamic.md), therefore you can configure a different set of buttons or preset values for each of the SiteAccesses in your installation.
```

Comment:
We use "SiteAccess-aware" throughout the rest of the doc, should be enough here:
```suggestion
Image Editor settings are SiteAccess-aware.
```